packages/webpack-inline-constant-exports-plugin/index.js

```diff
@@ -9,9 +9,9 @@ function addConstantExport( module, name, value ) {
 	module.constantExports[ name ] = value;
```

Comment:
The logic of this function can probably be replaced with `moduleGraph.getExportsInfo(module);`